althea_kernel_interface/src/hardware_info.rs

```diff
@@ -345,6 +349,70 @@ fn get_wifi_devices() -> Vec<WifiDevice> {
     ret
 }
 
+fn get_conntrack_info() -> Option<ConntrackInfo> {
+    const MAX_PATH: &str = "/proc/sys/net/netfilter/nf_conntrack_max";
+    const TABLE_COUNT_PATH: &str = "/proc/sys/net/netfilter/nf_conntrack_count";
+    const CURR_CONNS_PATH: &str = "/proc/net/nf_conntrack";
+
+    let curr_conn = Command::new("wc")
```

Comment:
instead of using command line commands can we read the file and count the number of lines that way? I actually believe rust has a function for this when given a file handle but maybe I'm wrong.

I'd like to avoid depending on programs like wc being installed. Some like ip are very hard to eliminate but I don't want to depend on things that we don't have to.

Reply:
Fixed